Frameworks/include/CoreTextInternal.h

```diff
@@ -32,38 +36,47 @@
 
 @interface _CTFrameSetter : NSObject {
 @public
-    idretaintype(_CTTypesetter) _typesetter;
+    StrongId<_CTTypesetter> _typesetter;
 }
 @end
 
 @interface _CTFrame : NSObject {
 @public
     _CTFrameSetter* _frameSetter;
     CGRect _frameRect;
+    StrongId<NSMutableArray> _lines;
+
+    // TODO::
+    // Do we need these anymore?
     CGSize _totalSize;
-    idretaintype(NSMutableArray) _lines;
     std::vector<CGPoint> _lineOrigins;
 }
 @end
 
 @interface _CTLine : NSObject {
 @public
-    NSRange _strRange;
+    CFRange _strRange;
+    CGPoint _lineOrigin;
+    StrongId<NSMutableArray> _runs;
```

Comment:
Dustin stopped by and looks like this is a bug with StongId

---
In reply to: [77873760](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77873760) [](ancestors = 77873760,77757580,77757492)